src/helpers/files.ts

```diff
@@ -86,7 +86,7 @@ export const generateFilesPaths = async (
     if (
       ignorePaths &&
       ignorePaths.some((ignorePath) =>
-        path.dirname(v).endsWith(`/${ignorePath}`),
+        path.dirname(v).includes(`/${ignorePath}`),
```

Comment:
includes has a side effect in this case, for example it'll match `src/ignored` and `src/ignored_with_adds`, this is the case because of which I didn't use includes. May be should use regexp or create normalizer for path string.

Reply:
u can try something like this:
```javascript
path.dirname(v).includes(`/${ignorePath}/`) || path.dirname(v).endsWith(`/${ignorePath}`),
```
or add `/` ending to includes left item
```javascript
`${path.dirname(v)}/`.includes(`/${ignorePath}/`),
```